src/plugins/skills/skill-guide-plugin.ts

```diff
@@ -76,11 +40,11 @@ function loadGuide(player: Player, guideId: number, subGuideId: number = 0, refr
         }
     }
 
-    const subGuide: SkillSubGuide = guide.subGuides[subGuideId];
+    const subGuide: SkillSubGuide = guide.sub_guides[subGuideId];
 
     player.modifyWidget(widgets.skillGuide, { childId: 1, text: (guide.name + ' - ' + subGuide.name) });
 
-    const itemIds: number[] = subGuide.lines.map(g => g.itemId).concat(new Array(30 - subGuide.lines.length).fill(null));
+    const itemIds: number[] = subGuide.lines.map(g => (g.item?.gameId || 0)).concat(new Array(30 - subGuide.lines.length).fill(null));
```

Comment:
I would like to mention that any skillguide with above 30 items will throw an exception, i.e. runecrafting skillguide -> Multiple subguide, because of the `new Array(30 - subGuide.lines.length).fill(null)` part

Reply:
Without that part, the client just shows gibberish, i.e.
![image](https://user-images.githubusercontent.com/6265839/104073040-006b0c00-520d-11eb-85cf-a1aceddc922e.png)

So I guess its a client-side limit?

Reply:
It's a widget limit, we might be able to change this limit in the future.